### ISSUE TITLE
MobileNet v3 Small Training Script

### ISSUE THREAD

**Issue by aidansmyth95ifx** [leondgarse/Keras_insightface#138]
Hi there.

Amazing work. I would love a training script example for MobilenetV3, particularly the small version. It has impressive latency and memory characteristics. Unfortunately, my LFW accuracy during training only hits 99.0%

**Comment by leondgarse** [comment]
Sorry, just saw this as closing issues... Are you still working on this?

Can't quit remember my results, just found some recording in my previous notebook:
- [mobilenet-on-emore](https://github.com/leondgarse/Atom_notebook/blob/master/public/2020/12-01_Insightface_training.md#mobilenet-on-emore)
- [mobilenet-arcface-and-curricularface-on-emore](https://github.com/leondgarse/Atom_notebook/blob/master/public/2021/04-07_insightface_advance.md#mobilenet-arcface-and-curricularface-on-emore)

It should be common reaching lfw accuracy of 99.7% referring to those results.
